Yank/experiment.py

```diff
@@ -368,7 +356,7 @@ def __init__(self, phases, number_of_iterations, switch_phase_interval):
 
     @property
     def iteration(self):
-        """Current iteration"""
+        """int, Current total number of iterations which have been run."""
```

Comment:
I don't think `@property` docstrings are accessible, so they will need to be documented in the class docstring.